### PR TITLE
chore: bump hubble sdk

### PR DESCRIPTION
**Pull request by bwanglzu**
In the latest jina-hubble-sdk, `notebook_login` has been merged into `login` by automatic detecting env

---

- [ ] This PR references an open issue
- [ ] I have added a line about this change to CHANGELOG

### CODE REVIEW

**Comment by bwanglzu** [comment]
do not merge until core updated

File: docs/walkthrough/login.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
 
 ```python
 import finetuner
 
-finetuner.login()  # use finetuner.notebook_login() in Jupyter notebook or Google Colab
+finetuner.login()
 ```
 
 A browser window should pop up with different login options.
-After {meth}`~finetuner.login()` or {func}`~finetuner.notebook_login()` you will see this in your terminal:
```
Comment (guenthermi) [inline docs/walkthrough/login.md:15]: It would be nice to have documentation for `login(interactive=True)`, but we can do it later

**Review by guenthermi — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ The following code snippet describes how to fine-tune ResNet50 on [Totally Looks
 import finetuner
 from finetuner.callback import EvaluationCallback
 
-finetuner.login()  # use finetuner.notebook_login() in Jupyter notebook/Google Colab
+finetuner.login()
```
Comment (LMMilliken) [inline README.md:136]:
```suggestion
finetuner.login() # use finetuner.login(interactive = True) in Jupyter notebook/Google Colab
```

File: docs/walkthrough/login.md (hunk 1 above)
Comment (bwanglzu) [reply docs/walkthrough/login.md:15]: it is actually not recommended, we implemented in the way that we can automatically detect a notebook @guenthermi

File: README.md [hunk 3]
```diff
@@ -154,12 +154,12 @@ Fine-tuning might take 5 minutes to finish. You can later re-connect your run wi
 ```python
 import finetuner
 
-finetuner.login()  # use finetuner.notebook_login() in Jupyter notebook or Google colab
+finetuner.login()
```
Comment (LMMilliken) [inline README.md:157]:
```suggestion
finetuner.login() # use finetuner.login(interactive = True) in Jupyter notebook/Google Colab
```

**Review by gmastrapas — COMMENTED** [review]
I dont see a bump in hubble? What about stubs and commons? should we bump?

File: docs/walkthrough/login.md [hunk 4]
```diff
@@ -2,17 +2,17 @@
...
 Since Finetuner leverages cloud resources for fine-tuning,
-you are required to {meth}`~finetuner.login()` (or {func}`~finetuner.notebook_login()`) and obtain a token from Jina before starting a fine-tuning job.
+you are required to {meth}`~finetuner.login()` and obtain a token from Jina before starting a fine-tuning job.
 It is as simple as:
 
 ```python
 import finetuner
 
-finetuner.login()  # use finetuner.notebook_login() in Jupyter notebook or Google Colab
+finetuner.login()
```
Comment (LMMilliken) [inline docs/walkthrough/login.md:11]:
```suggestion
finetuner.login() # use finetuner.login(interactive = True) in Jupyter notebook/Google Colab
```

File: docs/walkthrough/save-model.md [hunk 5]
```diff
@@ -21,7 +21,7 @@ In the example below, we show how to connect to an existing run and download a t
 ```python
 import finetuner
 
-finetuner.login()  # use finetuner.notebook_login() in Jupyter notebook or Google Colab
+finetuner.login()
```
Comment (LMMilliken) [inline docs/walkthrough/save-model.md:24]:
```suggestion
finetuner.login() # use finetuner.login(interactive = True) in Jupyter notebook/Google Colab
```

File: docs/walkthrough/login.md (hunk 1 above)
Comment (guenthermi) [reply docs/walkthrough/login.md:15]: ah ok good!

**Review by LMMilliken — APPROVED** [review]
LGTM, maybe these removed comments could just be reworded instead

**Comment by bwanglzu** [comment]
@gmastrapas that one has been taken care by #602

**Review by gmastrapas — APPROVED** [review]

**Pull request merged**